tests/Makefile

```diff
@@ -141,9 +177,9 @@ clean-gen:
 
 build:
 ifdef found_podman
-	podman build -t $(container_image) .
+	podman build --env CK8S_TESTS_BUILD=true --file tests/Dockerfile --tag $(container_image) ../
```

Comment:
Here you seem to be using `podman` but when checking if it exists you changed it to `docker --version`. Wouldn't `command -v podman` have better accuracy?

Reply:
I'll rework that check because I think I just wanted to shoehorn in to run docker first then podman but I didn't really think about different cases.